spec.bs

```diff
@@ -1509,18 +1521,38 @@ To <dfn>generate and score bids</dfn> given an [=auction config=] |auctionConfig
     to the result of running [=is debugging only in cooldown or lockout=] with |buyer|.
   1. [=map/For each=] |signalsUrl| → |perSignalsUrlGenerator| of |perSlotSizeQueryParam|:
     1. [=map/For each=] |slotSizeQueryParam| → |perSlotSizeQueryParam| of |perBuyerGenerator|:
+      1. Let |allTrustedBiddingSignals| be an [=ordered map=].
       1. Let |keys| be a new [=ordered set=].
       1. Let |igNames| be a new [=ordered set=].
+      1. Let |lengthLimit| be the maximum value of {{long}} interger.

```

Comment:
@domfarolino Is it ok to say "maximum value" like this?

Reply:
Yes I think this is fine. I'd word this:

```
Let |lengthLimit| be the maximum value that a {{long}} integer can hold.
```

(Note "integer" is currently spelled wrong in this PR)